src/Features/Core/Portable/InlineMethod/AbstractInlineMethodRefactoringProvider.cs

```diff
@@ -580,10 +580,20 @@ private async Task<SyntaxNode> GetChangedCallerAsync(Document document,
                 return declaredSymbol;
             }
 
-            if (IsFieldDeclarationSyntax(node)
-                && semanticModel.GetAllDeclaredSymbols(node, cancellationToken).SingleOrDefault() is IFieldSymbol fieldSymbol)
+            if (IsFieldDeclarationSyntax(node))
             {
-                return fieldSymbol;
+                foreach (var declarator in node.DescendantNodes().OfType<SyntaxNode>()
+                    .Where(n => _syntaxFacts.IsVariableDeclarator(n)))
```

Comment:
so this is a case where you want the descendent nodes to not descend in *once* you've hit an IsVariableDeclarator node.

Imagine you have something like `Action f = () => { var v = methodInvocation(); }`.  You'll hit both the `f = ...` declarator and the `v = ...` declarator.

Technically, not the end of the world here as you validate you're getting an IFieldSymbol.  But it's subtle and potentially a problem in the future.  so  better is to not descend any further once you hit the first layer of variable declarators.

does that make sense?